src/App/Entity/Talk.php

```diff
@@ -77,57 +77,58 @@ public static function fromStandardTalk(
         DateTimeImmutable $time,
         Speaker $speaker,
         string $title,
-        string $abstract
+        string $abstract,
```

Comment:
Not strictly true; in this context, we require an abstract, but the entity itself can exist without an abstract (e.g. "Opening comments"), not going in circles, just talking about different contexts I think :) by "resetting" it to null, we just have a consistent, expected state really